Directory.Build.targets

```diff
@@ -1,4 +1,8 @@
-<Project>
+<Project InitialTargets="SetupSemaphorePath">
+
+  <Target Name="SetupSemaphorePath">
+    <MakeDir Directories="$(CompletedSemaphorePath)" />
```

Comment:
`<WriteLinesToFile File="a/b/c/d.txt" />` seems to write the dirs fine, is it possible to switch to that and remove the pre-create targets?

Reply:
Ah, that doesn't change the modify date if the file already exists. This does though:

```
<WriteLinesToFile File="a/b/c/d.txt" Overwrite="true" />
```

Reply:
Thanks.  I'll change this.